admin-console/src/main/webapp/src/components/SystemBehaviorGroups/DeleteBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,59 @@
+import { ActionGroup, Button, Modal, ModalVariant, Spinner, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { BehaviorGroup } from '../../types/Notifications';
+
+interface DeleteBehaviorGroupModalProps {
+    onDelete: (systemBehaviorGroup?: BehaviorGroup) => Promise<boolean>;
+    bundleName?: string;
+    systemBehaviorGroupName?: string;
```

Comment:
These 2 should always be required. If we are going to delete something we should already know.